test/relations.test.js

```diff
@@ -2139,6 +2139,58 @@ describe('relations', function () {
     it('should set targetClass on scope property', function() {
       should.equal(Article.prototype.tagNames._targetClass, 'TagName');
     });
+
+    it('should apply inclusion fields to the target model', function(done) {
+      Article.create({title: 'a1'}, function (e, article) {
+        should.not.exist(e);
```

Comment:
Consider replacing `should.not.exist(e)` with `if (e) return done(e);`. The latter gives much more helpful error message.